src/ui/FieldTable/FieldTable.ts

```diff
@@ -133,13 +131,22 @@ export class FieldTable extends Component {
     }
 
     if (this.isTogglable()) {
-      this.toggleContainer = $$('div', { className: 'coveo-field-table-toggle-container' }).el;
+      const className = 'coveo-field-table-toggle-container';
+      this.toggleContainer = $$('div', { className, id: uniqueId(className) }).el;
```

Comment:
Hummmm, the `uniqueId` thingy is a thing for backbone js, so I'm not totally sure it's going to be doing the right thing.

Won't it generate `id=1, id=2, id=3` etc.. ? If so, we might have a problem as `1` for an ID is not as unique as we might think ;)

Reply:
It will add the number after whichever string we gave it, in this case the class name.
So it will generate the following: `coveo-field-table-toggle-container1`, `coveo-field-table-toggle-container2`, etc.

Reply:
Ah ! 👍